bigtable/client/client_options.h

```diff
@@ -92,6 +92,27 @@ class ClientOptions {
     channel_arguments_.SetGrpclbFallbackTimeout(fallback_timeout);
   }
 
+  /**
+   * Set the grpclb fallback timeout (std::chrono::duration<> duration) for the
+   * channel.
+   *
+   * Please see the docs for grpc::ChannelArguments::SetGrpclbFallbackTimeout()
+   * on https://grpc.io/grpc/cpp/classgrpc_1_1_channel_arguments.html
+   * for more details.
+   *
```

Comment:
Please document the `Rep` and `Period` template parameters.  Look here:

https://github.com/GoogleCloudPlatform/google-cloud-cpp/blob/master/bigtable/client/filters.h#L168

for an example.